k8s_api_client/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+k() {
+    kubectl --context kind-opa-kubernetes-api-client "$@"
+}
+
+kind create cluster --name opa-kubernetes-api-client --config kind-config.yaml
+
+# Install nginx ingress controller and wait for it to become ready
+k apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
```

Comment:
[nit] Safe to go with `master` here?

Reply:
Using a pinned version for the ingress would make sense if we could also use a pinned version for kind, but since we can't do that I think it makes sense assuming the latest version is used. This follows their own [docs on the topic](https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx).

Reply:
https://github.com/open-policy-agent/opa-envoy-plugin/blob/master/build/install-istio-with-kind.sh#L9-L13 👈 I'm not strongly opposed to using master here, but that's a an example where we're pinning a version of `kind`. But I might be missing something :smile:

Reply:
Yeah that would be an option of course, but I think it's fine assuming kind as a prerequisite for testing this, rather than do things like `sudo mv kind /usr/local/bin/` which I think is a bit intrusive for running a simple test - I'll add to the README though that kind and kubectl needs to be installed.

Reply:
:+1: thanks, I also prefer things that don't `sudo mv` anything 😅